packages/templates/react/.env.example

```
@@ -1 +1,4 @@
-VITE_BASE_URL="YOUR API URL"
\ No newline at end of file
+ VITE_BASE_URL="YOUR API URL"
```

dotenv-linter (3.3.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)